src/components/Atoms/Checkbox/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styled from "styled-components";
+import check from "../../../assets/svg/check.svg";
+import COLOR from "../../../variables/color";
+
+const Checkbox = (props) => {
+  return (
+    <SquareButton onClick={props.onClick}>

```

Comment:
JSのオブジェクトの分割代入を用いてこのようにもかけます。
どんなPropsを取るのかが明示されて可読性が上がると思うので、この先使ってみてください！
参考：　https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#%E3%82%AA%E3%83%96%E3%82%B8%E3%82%A7%E3%82%AF%E3%83%88%E3%81%AE%E5%88%86%E5%89%B2%E4%BB%A3%E5%85%A5
```suggestion
const Checkbox = ({onClick}) => {
  return (
    <SquareButton onClick={onClick}>

```